internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -96,6 +98,15 @@ COPY roles/ ${HOME}/roles/[[ end ]]
 [[- if .Playbook ]]
 COPY playbook.yml ${HOME}/playbook.yml[[ end ]]
 
+[[- if or .Roles .Playbook ]]
```

Comment:
Not positive, but you might be able to simplify this to `[[- if or (or .Roles .Playbook) .Watches ]]` so you don't have to repeat the RUN command. Yay for [prefix notation](https://en.wikipedia.org/wiki/Polish_notation)!